getgist/__main__.py

```diff
@@ -48,6 +48,11 @@
     Usage:  `lsgists <GitHub username>`.
 """
 
+LSMY_DESC = """
+    Call `lsgists` files assuming the user is set in an envvar called GETGIST_USER.
+    Usage:  `lsgists <GitHub username>`.
```

Comment:
Actually this line 53 doesn't make sense in this command helm text. Why suggesting `lsgists` here?

```suggestion
    See `lsgists --help` for more details.
```

Reply:
Can you fix that too? Or did I get it wrong?

Reply:
Yes, I can fix it! Totally forgot about it! LOL

Reply:
Sorry, but 741256b  is not what I have suggested, and it actually breaks a pattern.

* `getmy --help` ends with _See `getgist --help` for more details._
* `putmy --help` ends with _See `putgist --help` for more details._

So I expected (as in my first comment) `mygists --help` to end with _See `lsgists --help` for more details._.

Does that make sense?

Reply:
Makes lots of sense! Going to update it!